File: Pset1/Cash.c
#include <stdio.h>
#include <math.h>
#include <cs50.h>

int main(void) 
{
    float money;
    int roundMoney;
    int counter = 0;

    do
    {
        money = get_float("Enter with the money value: ");
    }
    while (money <= 0);

    money = money * 100;    
    roundMoney = round(money);
    
    while (roundMoney - 25 >= 0)
    {
        roundMoney = roundMoney - 25;
        counter++;
    }
    while (roundMoney - 10 >= 0)
    {
        roundMoney = roundMoney - 10;
        counter++;
    }    
    while (roundMoney - 5 >= 0)
    {
        roundMoney = roundMoney - 5;
        counter++;
    }
    while (roundMoney - 1 >= 0)
    {
        roundMoney = roundMoney - 1;
        counter++;
    }
   
    printf("%i\n", counter);
}

File: Pset6/Cash/Cash.py
from cs50 import get_float

while True:
    change = get_float("Enter with the money value: ")
    if change >= 0:
        break

change *= 100
counter = 0

while change - 25 >= 0:
    change -= 25
    counter += 1

while change - 10 >= 0:
    change -= 10
    counter += 1

while change - 5 >= 0:
    change -= 5
    counter += 1

while change - 1 >= 0:
    change -= 1
    counter += 1

print(counter)